### DISCUSSION TITLE
The charon process has a segmentation fault, generating core

### DISCUSSION THREAD

**Discussion by S-rh** [strongswan/strongswan#2385] [Q&A] [answered]
Use strongswan5.5.1 to establish the IKV1 central node, and continue to run the flow in the flow trigger mode. After running for one hour, the core file appears.
Can you help me find out what the problem is?

[gdb.txt](https://github.com/user-attachments/files/16490438/gdb.txt)

**Comment by tobiasbrunner** [comment, marked as answer]
Sorry, but strongSwan 5.5.1 is way too old to provide support for (was released nearly 8 year ago). Please update to a newer version.

**Reply by S-rh** [reply]
OK, thank you very much

**Reply by siddeshpawar** [reply]
did you figured it out how to fix it , i had done multiple installation and rebuilds still no luck
